data-serving/data-service/schemas/cases.schema.json

```diff
@@ -16,25 +16,19 @@
         "bsonType": "object",
         "additionalProperties": false,
         "properties": {
-          "age": {
+          "ageRange": {
             "bsonType": "object",
             "additionalProperties": false,
             "properties": {
-              "range": {
-                "bsonType": "object",
-                "additionalProperties": false,
-                "properties": {
-                  "start": {
-                    "bsonType": "double",
-                    "minimum": -1,
-                    "maximum": 200
-                  },
-                  "end": {
-                    "bsonType": "double",
-                    "minimum": -1,
-                    "maximum": 200
-                  }
-                }
+              "start": {
+                "bsonType": "double",
+                "minimum": -1,
+                "maximum": 200
```

Comment:
turtles can live up to almost 300 years!! ;-)
Kidding but I saw "species" before somewhere so if we're talking non-humans we can raise that limit a bit.

Reply:
done, will fix later in the mongoose validator :)